src/main/java/org/symphonyoss/integration/webhook/salesforce/SalesforceWebHookIntegration.java

```diff
@@ -66,6 +74,18 @@ public void onConfigChange(IntegrationSettings settings) {
 
   @Override
   public String parse(WebHookPayload input) throws WebHookParseException {
+    String messageML;
+
+    if (isContentTypeJSON(input)) {
+      try {
+        messageML = processIntegrationWithJSON(input);
```

Comment:
You can deal with IOException inside the method. 

Something like that: 

```
if (isContentTypeJSON(input)) {
  return processIntegrationWithJSON(input);
}
```

Inside the 'processIntegrationWithJSON' you should throw the parse exception.